stdlib/private/SwiftPrivate/PRNG.swift

```diff
@@ -14,19 +14,19 @@ import SwiftShims
 
 @inline(never) @_semantics("stdlib_binary_only") // Hide the libbsd dependency
 public func rand32() -> UInt32 {
-  return _swift_stdlib_arc4random()
+  return _swift_stdlib_cxx11_mt19937()
 }
 
 @inline(never) @_semantics("stdlib_binary_only") // Hide the libbsd dependency
 public func rand32(exclusiveUpperBound limit: UInt32) -> UInt32 {
-  return _swift_stdlib_arc4random_uniform(limit)
+  return _swift_stdlib_cxx11_mt19937_uniform(limit)
 }
 
 @inline(never) @_semantics("stdlib_binary_only") // Hide the libbsd dependency
```

Comment:
I think you can drop `@inline(never) @_semantics("stdlib_binary_only")` from these three functions now, these attributes are not serving any purpose now that we don't depend on libbsd.